### PR TITLE
k9s: update to 0.25.15

### PR DESCRIPTION
**Pull request by ScOut3R**
**Testing the changes**

- I tested the changes in this PR: YES

### CODE REVIEW

File: srcpkgs/k9s/template [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # Template file for 'k9s'
 pkgname=k9s
-version=0.25.8
+version=0.25.12
```
Comment (ericonr) [inline srcpkgs/k9s/template:3]: .14 is out

Comment (ScOut3R) [reply srcpkgs/k9s/template]: thank you for noticing @ericonr, those guys are fast, I've updated the PR

Comment (ericonr) [reply srcpkgs/k9s/template]: Right? So many releases!

Comment (ericonr) [reply srcpkgs/k9s/template]: Also lmao .15 is now out.

Comment (ScOut3R) [reply srcpkgs/k9s/template]: They don't sleep for sure. I've updated the PR.

**Pull request merged**